diff --git a/config/mainnet/env.toml b/config/mainnet/env.toml
index 4d71a824e..89d82d0fe 100644
--- a/config/mainnet/env.toml
+++ b/config/mainnet/env.toml
@@ -54,7 +54,7 @@ withdrawal_limit = 500_000000
 withdrawal_waiting_period = 1800
 minimum_deposit = 1_000000
 minimum_shares_dust_threshold = 1_000
-performance_fee_bps = 1500
+performance_fee_bps = 2000
 management_fee_bps = 100
 vault_name = "Maple SyrupUSDC"
 vault_operator = "51BTctdgWNB9ypgRMJ12NxTW6ZQvWBwNDKRsPHjhuj1N" # Squads multisig signer
